webp/view/src/main/cpp/webp_decode.cpp

```diff
@@ -111,6 +112,7 @@ void WebpDecoder::DecodeFrameInternal() {
             break;
         default:
             assert( 0 );
+            delete  [] buf;
```

Comment:
sample is under debug build, assert should always happen; under release build the path is not addressed well: in some samples we are trying NOT to crash the things ( it might end up does nothing )